Installing-and-Configuring/OS/index.md

```diff
@@ -58,17 +58,17 @@ Your other partitions may be encrypted or not based on your preferences and this
 
 ### Using TPM for decryption of drives
 
-Heads can seal a Disk Unlock Key using the TPM.  This will ensure that upon booting only a disk unlock passphrase and the proper PCR state can access the data on the root partition.  This approach uses a Security Dongle for disk decryption and may be used to lock the encrypted data to the hardware.
+Heads can protect a LUKS Disk Unlock Key using the TPM.  This will ensure that upon booting with Heads only a TPM disk unlock passphrase and the proper PCR state can access the data on the root partition.  In this model the TPM enforces rate limits to prevent brute force attacks.  If the disk is moved to a different system the [LUKS disk unlock passphrase/recovery key](/Keys/#tpm-disk-encryption-key) in slot 0 may be used to access the encrypted data.  This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
 
-If this feature is enabled, heads will ask for a Disk Unlock Passphrase when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The Disk Unlock passphrase will be used to generate a Disk Unlock Key to be placed in the second LUKS keyslot for the root partition.  The disk unlock key is sealed in the TPM along with measurements taken at that time.
+If the Heads TPM drive decryption feature is enabled, Heads will [ask for](/Keys/#tpm-disk-encryption-key) a Disk Unlock Passphrase (which might be a numerical PIN) when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The TPM Disk Unlock passphrase will be used to generate a LUKS Disk Unlock Key to be placed in keyslot 1 for the root partition.  The LUKS disk unlock key is sealed in the TPM along with measurements taken at that time.
 
-On boot, heads will ask for the disk unlock passphrase that will release the Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the Key was created.  
+On boot, Heads will ask for the TPM disk unlock passphrase that will release the LUKS Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the keys were created.  
```

Comment:
On boot, Heads will ask for the TPM Disk Unlock Key passphrase that will release the LUKS Disk Unlock Key from TPM NV reserved memory. This will only succeed if the current measurements (LUKS header, firmware measurements, Disk Unlock Key passphrase) match the ones taken when the keys were created.

Reply:
updated.  push in a minute